website/files/models/osfstorage.py

```diff
@@ -33,6 +34,25 @@ def get_or_create(cls, node, path):
         # Dont raise anything a 404 will be raised later
         return cls.create(node=node, path=path)
 
+    @classmethod
+    def get_file_guids(cls, materialized_path, provider, guids, node=None):
+        path = materialized_path
```

Comment:
Minor, but it looks like you could probably just strip the `/` here and make the two lines below a tiny bit more readable.